test/cpp/jit/test_gpu.cpp

```diff
@@ -7486,7 +7486,7 @@ TEST(NVFuserTest, FusionInputsIdLookup_CUDA) {
   at::Tensor t2 = at::randn({6, 4}, options);
 
   // create a cache with max size 2;
-  auto inputs_id_lookup = InputsIdLookup(2);
+  torch::jit::fuser::cuda::InputsIdLookup inputs_id_lookup(2);
```

Comment:
what is the reason for this change?

Reply:
I added a mutex in the struct so it's not copyable. We don't really need copy so I'm keeping it simple.